tools/site_and_regional/plumber2_usermods.py

```diff
@@ -49,7 +49,7 @@ def write_usermods(lat,lon,site,start_year,end_year,
         '\n' \
         # TODO, get working for CTSM5.1, remove this line as it's redundant after PLUMBER2SITE is added
         # Alternatively, we can take this out of default/user_nl_clm since doing it this way is works fine TODO for 5.2
-        'echo "fsurdat=\'/glade/u/home/wwieder/CTSM/tools/site_and_regional/subset_data_single_point/surfdata_1x1_PLUMBER2_'+site+'_hist_16pfts_Irrig_CMIP6_simyr2000_c231005.nc \' " >> user_nl_clm \n' \
+        'echo "fsurdat=\'\$DIN_LOC_ROOT/lnd/clm2/surfdata_esmf/PLUMBER2/surfdata_1x1_PLUMBER2_'+site+'_hist_2000_16pfts_c240326.nc\'  " >> user_nl_clm \n' \
```

Comment:
We can likely remove this all from plumber2_usermods.py script, if we add PLUMBER2SITE to cdeps this can be handled in user_nl_clm in the default usermods_dirs.

Reply:
Updating this in the commit titled "update usermods script and shell commands for fsurdat"